src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml.cs

```diff
@@ -34,7 +34,7 @@ public string Message
 
         public MessageType MessageType
         {
-            get { return (MessageType)GetValue(MessageProperty); }
+            get { return (MessageType)GetValue(MessageTypeProperty); }
```

Comment:
This was previously set to `MessageProperty`, which didn't seem right, given the property type. Nothing looked like it was breaking because of it, and it doesn't look like anything broke after changing it.